### PR TITLE
kicad 5.1

### PR DESCRIPTION
**Pull request by eszlari**

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64", "i386"]
```
Comment (TingPing) [inline flathub.json:2]: This is obviously a regression.

**Comment by eszlari** [comment]
Both ARM builds were successful, but failed at `8 Generate deltas`.
```
flatpak build-update-repo --generate-static-deltas --static-delta-ignore-ref=*.Debug  --static-delta-ignore-ref=*.Sources repo
 in dir /home/builder/worker/build-aarch64-7/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak build-update-repo --generate-static-deltas --static-delta-ignore-ref=*.Debug  --static-delta-ignore-ref=*.Sources repo'
 using PTY: False
Updating appstream branch
Generating static deltas
Generating delta: appstream2/aarch64 (ebd77be73f)
Generating delta: app/org.kicad_pcb.KiCad/aarch64/test (e993388cf6)
Generating delta: runtime/org.kicad_pcb.KiCad.Locale/aarch64/test (a9549d1d1e)
command timed out: 1200 seconds without output running b'flatpak build-update-repo --generate-static-deltas --static-delta-ignore-ref=*.Debug  --static-delta-ignore-ref=*.Sources repo', attempting to kill
process killed by signal 9
program finished with exit code -1
elapsedTime=1202.442146
```

**Comment by eszlari** [comment]
@TingPing Could this failure be connected to the large size of `kicad-packages3D`:

> Total Download Size: 291.53 MiB
> Total Installed Size: 4449.22 MiB

https://github.com/KiCad/kicad-packages3D/issues/369

Do you think the ARM builds should block the update?

**Comment by eszlari** [comment]
@TingPing Ping?

**Comment by eszlari** [comment]
Still fails for arm/aarch64 in `Generate deltas` step. @alexlarsson

@TingPing @nedrichards @barthalion Should we just skip these arches for now, to finally get a up-to-date version in.

**Comment by bilelmoussaoui** [comment]
@eszlari go ahead. Can you please open an issue to track the arm/aarch64 build too?

**Comment by eszlari** [comment]
> Can you please open an issue to track the arm/aarch64 build too?

https://github.com/flathub/org.kicad_pcb.KiCad/issues/19

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by nedrichards** [comment]
bot, build

**Comment by eszlari** [comment]
```
flatpak-builder --show-manifest org.kicad_pcb.KiCad.json
 in dir /srv/buildbot-master/master/workers/MasterWorkerUntrusted/Builds/build (timeout 1200 secs)
 watching logfiles {}
 argv: [b'flatpak-builder', b'--show-manifest', b'org.kicad_pcb.KiCad.json']
 using PTY: False
(flatpak-builder:13011): flatpak-builder-ERROR **: 14:48:06.548: Failed to load included manifest (/srv/buildbot-master/master/workers/MasterWorkerUntrusted/Builds/build/shared-modules/glu/glu-9.0.0.json): Failed to open file “/srv/buildbot-master/master/workers/MasterWorkerUntrusted/Builds/build/shared-modules/glu/glu-9.0.0.json”: No such file or directory
process killed by signal 5
program finished with exit code -1
elapsedTime=0.058376
```
I had the same error with OBS and KDE runtime 5.13 (19.08 based).

Why do shared-modules and 19.08 not work together? Is this a problem with the build server?

Edit: fixed by https://github.com/flathub/buildbot/commit/1b28123536096d2a6f898b08972a1e30f98f3925

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
Thanks!

**Pull request merged**